saleor/core/utils/translations.py

```diff
@@ -51,3 +45,9 @@ def get_translated_keys(self) -> Dict[str, Any]:
 
     def get_translation_context(self) -> Dict[str, Any]:
         return {}
+
+
+def get_translation(instance, language_code) -> TranslationWrapper:
+    if not language_code:
+        language_code = settings.LANGUAGE_CODE
```

Comment:
I'm wondering if this is needed. IMO we should set translations only when explicitly requested, in this case, when `language_code` is known, so when it was provided in checkout mutations. In the cloud, the language code is always `en`. I think we could return `None` when language code is not provided.